core/modules/system/system.module

```diff
@@ -928,7 +935,7 @@ function system_menu() {
   );
   $items['admin/config/system/site-information'] = array(
     'title' => 'Site information',
-    'description' => 'Change site name, logo, favicon, slogan, e-mail address, default front page, and number of posts per page, error pages.',
+    'description' => 'Change site name, logo, favicon, slogan, and from e-mail address.',
```

Comment:
The home page setting can still be configured here + while at it, let's reorder these to reflect their order in the form.

```suggestion
    'description' => 'Change site name and slogan, from e-mail address, logo, favicon, and default home page.',
```